x-pack/plugins/cases/public/components/create/index.test.tsx

```diff
@@ -54,24 +52,16 @@ const useGetIssueTypesMock = useGetIssueTypes as jest.Mock;
 const useGetFieldsByIssueTypeMock = useGetFieldsByIssueType as jest.Mock;
 const fetchTags = jest.fn();
 
-const fillForm = (wrapper: ReactWrapper) => {
-  wrapper
-    .find(`[data-test-subj="caseTitle"] input`)
-    .first()
-    .simulate('change', { target: { value: sampleData.title } });
-
-  wrapper
-    .find(`[data-test-subj="caseDescription"] textarea`)
-    .first()
-    .simulate('change', { target: { value: sampleData.description } });
-
-  act(() => {
-    (
-      wrapper.find(EuiComboBox).at(0).props() as unknown as {
-        onChange: (a: EuiComboBoxOptionOption[]) => void;
-      }
-    ).onChange(sampleTags.map((tag) => ({ label: tag })));
-  });
+const fillForm = async (renderer: RenderResult) => {
```

Comment:
**nit:** not sure this needs to be a function 😄

the function is only used once and these`findByTestId` seem very specific.